R/listLearners.R

```diff
@@ -75,6 +83,8 @@ listLearners.character  = function(obj, properties = character(0L),
   # make really sure we filter out our own mock learners here, they have a very unique name
   meths = meths[!grepl("__mlrmocklearners__", meths)]
   res = err = vector("list", length(meths))
+  res.table = data.frame()
+  res.table.props = c(numerics = FALSE, factors = FALSE, ordered = FALSE, missings = FALSE, weights = FALSE, twoclass = FALSE, multiclass = FALSE, class.weights = FALSE, prob = FALSE, se = FALSE, lcens = FALSE, rcens = FALSE, icens = FALSE)
   learner.classes = vcapply(strsplit(meths, "makeRLearner\\."), function(x) x[2L])
   if (!create && check.packages) {
     installed.packs = rownames(installed.packages())
```

Comment:
We should not call `installed.packages()` (think of large libraries), but use `find.package()` instead.